GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/MainWindowViewModel.cs

```diff
@@ -110,7 +110,39 @@ public MainWindowViewModel()
             CreateTimer();
             LoadCommands();
 
-            ActiveUsers.CollectionChanged += ActiveUsersChanged;
+            AllUsers.CollectionChanged += AllUsersCollectionChanged;
+        }
+
+        private void AllUsersCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
```

Comment:
Could you use a more descriptive var name than `e`? Maybe something like `changedEventArgs`?